# HG changeset patch
# User jwe
# Date 750572775 0
#      Thu Oct 14 04:26:15 1993 +0000
# Node ID be50d501b2a77d603fbb28ec18298869cb770678
# Parent  abc5f0a0785f137daf3d0e2c8b07ae42b827457f
[project @ 1993-10-14 04:26:15 by jwe]
(decode_prompt_string): ctime returns time_t, which might not be a
long.

diff --git a/src/utils.cc b/src/utils.cc
--- a/src/utils.cc
+++ b/src/utils.cc
@@ -690,17 +690,17 @@ decode_prompt_string (char *string)
 		c = 0;
 		goto add_string;
 	      }
 	  
 	    case 't':
 	    case 'd':
 	      /* Make the current time/date into a string. */
 	      {
-		long the_time = time (0);
+		time_t the_time = time (0);
 		char *ttemp = ctime (&the_time);
 		temp = strsave (ttemp);
 
 		if (c == 't')
 		  {
 		    strcpy (temp, temp + 11);
 		    temp[8] = '\0';
 		  }
